packages/components-js/tests/unit/fixtures/stats.json

```diff
@@ -132,12 +132,14 @@
   {
     "chunkShortName": "scroller",
     "size": 7373,
-    "gzipSize": 2913
+    "gzipSize": 2915
   },
   {
     "chunkShortName": "segmented-control",
     "size": 7214,
     "gzipSize": 2827
+    "size": 7760,
+    "gzipSize": 3071
```

Comment:
Looks like something went wrong here while merging.

Reply:
true, fixed it